variants/tracker-t1000-e/platformio.ini

```diff
@@ -0,0 +1,15 @@
+; tracker-t1000-e  v0.9.0
+[env:tracker-t1000-e]
+extends = nrf52840_base
+board = tracker-t1000-e
+board_level = extra
+; platform = https://github.com/maxgerhardt/platform-nordicnrf52#cac6fcf943a41accd2aeb4f3659ae297a73f422e
+build_flags = ${nrf52840_base.build_flags} -Ivariants/tracker-t1000-e -DTRACKER_T1000_E  -DRADIOLIB_GODMODE
```

Comment:
Does this need  -Isrc/platform/nrf52/softdevice -Isrc/platform/nrf52/softdevice/nrf52 ?

Reply:
changed it to the softdevice now